src/compass/utils/helpers.py

```diff
@@ -219,7 +219,7 @@ def _convert_to_utm(points_xy, epsg_src, epsg_dst):
     return list(zip(xt, yt))
 
 
-def get_burst_bbox(burst_id, burst_db_file=None, burst_db_conn=None):
+def burst_bbox_from_db(burst_id, burst_db_file=None, burst_db_conn=None):
```

Comment:
The sql is erroring for me right now, and I see two ways to fix it:

1. run the query multiple times as it was
2. Adjust the code to something like
```python
    burst_ids = [burst_id] if isinstance(burst_id, str) else burst_id

    placeholder = ','.join('?' * len(burst_ids))
    query = f"SELECT epsg, xmin, ymin, xmax, ymax, burst_id_jpl FROM burst_id_map WHERE burst_id_jpl in ({placeholder})"
    cur = burst_db_conn.execute(query, burst_ids)
```

option 2 would be if you really want only 1 sql query to run, since I dont think there's another way to do a `WHERE ... IN ` to select multiple matching items without building up a bunch of `?, ?, ?,...`
https://stackoverflow.com/questions/1309989/parameter-substitution-for-a-sqlite-in-clause
i'm also not sure how much time saving it would be? when I do the single SQL where with the `WHERE ... IN` clause for 10 burst ids, it's 109 microseconds.
When I do it with 10 sql queries, it's about 200 microseconds.